### ISSUE TITLE
[UI] Show task actions while viewing live log

### ISSUE THREAD

**Issue by reuben** [taskcluster/taskcluster#1960]
Would be nice to be able to trigger task actions (e.g. re-run) while viewing the live log, without having to go back to the task viewer first.

**Comment by naima-shk** [comment]
@helfi92  Can i try this one please ?

**Comment by naima-shk** [comment]
This seems more like new feature or enhancement than a bug.

**Comment by reuben** [comment]
Yes, sorry, I used the bug template because I missed the option to file a blank issue. I can't take the bug label out though.

**Comment by naima-shk** [comment]
@reuben  No problem . It would be great  if you provide more details about the implementation of new feature or enhancement? :)

**Comment by reuben** [comment]
Here's an example of a task: https://community-tc.services.mozilla.com/tasks/co7YayJNRMOD98mziRYDHQ

When I click to view the live log: https://community-tc.services.mozilla.com/tasks/co7YayJNRMOD98mziRYDHQ/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2Fco7YayJNRMOD98mziRYDHQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log

In the buttons on the bottom right I can only go back to the task, view a different task, or get the raw log. I'd like to also have the actions from the task itself (rerun, retrigger, cancel) in the menu of the live log view.

**Comment by Rolikasi** [comment]
It looks interesting!) 
I can try to add new features

**Comment by Rolikasi** [comment]
@helfi92, do we need to add all buttons to Log view or only `cancel`, `rerun` and `retrigger`?

**Comment by helfi92** [comment]
All :)

**Comment by Rolikasi** [comment]
All buttons are rendered in views/Tasks/ViewTask/index.jsx
Log is rendered in views/Tasks/TaskLog/index.jsx
We can make a new component with these buttons instead of duplicate them in log. Is it the right way?

**Comment by helfi92** [comment]
> We can make a new component with these buttons instead of duplicate them in log. Is it the right way?

💯
